deploy/examples/with-ingress-tls.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: jaegertracing.io/v1
```

Comment:
I think only one example file with both properties is enough.

Reply:
What would you prefer:

1. Keep the `with-ingress-host-and-tls` example, remove `with-ingress-host` and `with-ingress-tls`
2. Add ingress options (basically `with-ingress-host-and-tls`): 
    ```
      ingress:
        hosts:
          - jaeger.example.com
        tls:
          - hosts:
              - jaeger.example.com
            secretName: jaeger-example-com-tls
    ```
    to `all-in-one-with-options` and remove all `with-ingress-*` examples (`with-ingress-host-and-tls`, `with-ingress-host`, `with-ingress-tls`

Reply:
Keeping `with-ingress-host-and-tls` would be fine

Reply:
Done